lib/puppet/face/help.rb

```diff
@@ -66,7 +66,7 @@
       facename, actionname = args
       if legacy_applications.include? facename then
         if actionname then
-          raise ArgumentError, "Legacy subcommands don't take actions"
+          raise ArgumentError, _("Legacy subcommands don't take actions")
```

Comment:
That may be true, but I think we want to avoid changing the content of existing strings as much as possible. Too hard to tell who might be relying on what (including us in our tests). In this case it probably doesn't matter, but I think I'll still leave it as-is till I hear otherwise from the people in touch with the translators.